modules/bamtools/convert/main.nf

```diff
@@ -0,0 +1,36 @@
+process BAMTOOLS_CONVERT {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::bamtools=2.5.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bamtools:2.5.1--h9a82719_9' :
+        'quay.io/biocontainers/bamtools:2.5.1--h9a82719_9' }"
+
+    input:
+    tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("*.{bed,fasta,fastq,json,pileup,sam,yaml}"), emit: out
```

Comment:
Could change the output var name - this would be written in a pipeline as BAMTOOLS_CONVERT.out.out

Reply:
Of course, I changed it to `data`